### PR TITLE
RSWEB-7868: Make region selector great again

### PR DESCRIPTION
**Pull request by WidgetsBurritos**
This PR fixes the region selector on mobile by pushing the text into multiple lines instead of trying to stack them in a small space.

### CODE REVIEW

File: styleguide/_themes/derek/scss/globalElements/basement.scss [hunk 1]
```diff
@@ -26,7 +26,7 @@
 
 .basement-copyright {
   color: $gray-base;
-  text-align: right;
+  text-align: center;
```
Comment (brantdarilek) [inline styleguide/_themes/derek/scss/globalElements/basement.scss]: Moving copyright to the center of the page?

Comment (WidgetsBurritos) [inline styleguide/_themes/derek/scss/globalElements/basement.scss]: just on mobile.  See the media query below.

**Comment by amberRrucker** [comment]
Very clean solution. +1

**Comment by brantdarilek** [comment]
👍

**Pull request merged**